spec.bs

```diff
@@ -492,8 +513,26 @@ The <dfn element-attr for=fencedframe>allow</dfn> attribute, when specified, det
 in the <{fencedframe}>'s [=fenced navigable container/fenced navigable=] is initialized. Its value
 must be a [=serialized permissions policy=]. [[!PERMISSIONS-POLICY]]
 
-The IDL attribute <dfn attribute for=HTMLFencedFrameElement>allow</dfn> must [=reflect=] the
-respective content attribute of the same name.
+The <dfn element-attr for=fencedframe>sandbox</dfn> attribute, when specified, enables a set of
+extra restrictions on any content hosted by the <{fencedframe}>. Its value must be an [=unordered
+set of unique space-separated tokens=] that are [=ASCII case-insensitive=]. The allowed values are
+the same as the allowed values for the <{iframe}>'s <{iframe/sandbox}> attribute.
+
+When an <{fencedframe}> element's <{fencedframe/sandbox}> attribute is set or changed while it has a
+non-null [=fenced navigable container/fenced navigable=], the user agent must [=parse a sandboxing
+directive=] given the attribute's value and the <{fencedframe}> element's [=fencedframe/fencedframe
+sandboxing flag set=].
+
+When an <{fencedframe}> element's <{fencedframe/sandbox}> attribute is removed while it has a
```

Comment:
Same as above, I think we can cover the removal case with the attribute changed steps too, right?

Reply:
Added a section to handle removal. I'm assuming that attribute removal will invoke `[=attribute change steps=]` with `|value|` = null, but correct me if that assumption is incorrect.